src/cogs/errorhandler.py

```diff
@@ -188,8 +188,6 @@ async def on_command_error(self, ctx: Context, error: Exception):
 
             elif isinstance(error, numpy.linalg.LinAlgError):
                 return await ctx.error("The given warp points are unsolvable.")
-            elif isinstance(error, sqlite3.OperationalError):
-                return await ctx.error(f"There was an error in your SQL.\n`{error}`")
```

Comment:
This seems like it shouldn't be removed

Reply:
incorrect sql mostly comes from internal errors, so it's more useful to get a traceback

Reply:
Sure.